presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveCsvStorageFormat.java

```diff
@@ -0,0 +1,176 @@
+    }
+
+    private void testCreateCsvTableAs(String tableName, String additionalParameters)
+    {
+        query("DROP TABLE IF EXISTS " + tableName);
+
+        query(format(
+                "CREATE TABLE %s WITH (format='CSV' %s) AS " +
+                        "SELECT " +
+                        "cast(linestatus AS varchar) AS linestatus, cast(shipmode AS varchar) AS shipmode, cast(returnflag AS varchar) AS returnflag " +
+                        "FROM tpch.tiny.lineitem",
+                tableName,
+                additionalParameters));
+
+        assertSelect("select max(linestatus), max(shipmode), count(returnflag) from %s", tableName);
+
+        query("DROP TABLE " + tableName);
+    }
+
+    @Test(groups = STORAGE_FORMATS)
+    public void testInsertIntoPartitionedCsvTable()
+    {
+        testInsertIntoPartitionedCsvTable("storage_formats_test_insert_into_partitioned_csv", "");
+    }
+
+    @Test(groups = STORAGE_FORMATS)
+    public void testInsertIntoPartitionedCsvTableWithCustomProperties()
+    {
+        testInsertIntoPartitionedCsvTable("storage_formats_test_insert_into_partitioned_csv_with_custom_properties", ", csv_escape = 'e', csv_separator='s', csv_quote='q'");
+    }
+
+    private void testInsertIntoPartitionedCsvTable(String tableName, String additionalParameters)
+    {
+        query("DROP TABLE IF EXISTS " + tableName);
+
+        query(format(
+                "CREATE TABLE %s(" +
+                        "   linestatus    VARCHAR," +
+                        "   shipinstruct  VARCHAR," +
+                        "   shipmode      VARCHAR," +
+                        "   comment       VARCHAR," +
+                        "   returnflag    VARCHAR," +
+                        "   suppkey       BIGINT" +
+                        ") WITH (format='CSV' %s, partitioned_by = ARRAY['suppkey'])",
+                tableName,
+                additionalParameters));
+
+        query(format(
+                "INSERT INTO %s " +
+                        "SELECT " +
+                        "linestatus, shipinstruct, shipmode, comment, returnflag, suppkey " +
+                        "FROM tpch.%s.lineitem", tableName, TPCH_SCHEMA));
+
+        assertSelect("select max(linestatus), max(shipinstruct), max(shipmode), max(suppkey) from %s", tableName);
+
+        query("DROP TABLE " + tableName);
+    }
+
+    @Test(groups = STORAGE_FORMATS)
+    public void testCreatePartitionedCsvTableAs()
+    {
+        testCreatePartitionedCsvTableAs("storage_formats_test_create_table_as_select_partitioned_csv", "");
+    }
+
+    @Test(groups = STORAGE_FORMATS)
+    public void testCreatePartitionedCsvTableAsWithCustomParamters()
+    {
+        testCreatePartitionedCsvTableAs(
+                "storage_formats_test_create_table_as_select_partitioned_csv_with_custom_parameters",
+                ", csv_escape = 'e', csv_separator='s', csv_quote='q'");
```

Comment:
nit: Add spaces or remove the first `csv_escape='e'`
```suggestion
                ", csv_escape = 'e', csv_separator = 's', csv_quote = 'q'");
```

Reply:
@kokosing bump